Sources/SwifterSwift/CoreLocation/CLVisitExtensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CLVisitExtensions.swift
+//  SwifterSwift
+//
+//  Created by Trevor Phillips on 09/01/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+#if canImport(CoreLocation) && targetEnvironment(macCatalyst)
```

Comment:
Is this check correct? It only works on macCatalyst?

Reply:
The correct check, shouldn't be `&& !targetEnvironment(macCatalyst)`?

Reply:
Yeah I caught that.. I was just about to fix it. Oops. It's on ios and mac catalyst